src/utils/helpers/index.ts

```diff
@@ -290,3 +290,12 @@ export function getMetaObject(sdkDetails: any, metaData: any, navigator: any) {
   };
   return metaObject;
 }
+
+export function isValidURL(urlString: string): boolean {
```

Comment:
there is already existing isValidURL function to validate URL. please use the same.